src/synthesizer/emission_models/stellar/models.py

```diff
@@ -78,24 +80,42 @@ class LineContinuumEmission(StellarEmissionModel):
     def __init__(
         self,
         grid,
-        label="linecont",
+        fesc_ly_alpha,
+        label="line_emission",
         fesc=0.0,
         **kwargs,
     ):
         """
-        Initialise the LineContinuumEmission object.
+        Initialise the LineContributionEmission object.
 
         Args:
             grid (synthesizer.grid.Grid): The grid object to extract from.
+            fesc_ly_alpha (float): The escape fraction of Lyman-alpha.
             label (str): The label for this emission model.
             fesc (float): The escape fraction of the emission.
         """
+        print(fesc, fesc_ly_alpha)
+        # Create the child models
+        linecont_no_lyman = StellarEmissionModel(
```

Comment:
This the important bit, automatically defining the 2 contributions with their unique escape fractions.